tests/test_api.py

```diff
@@ -85,7 +85,7 @@ def test_serve_swagger_ui(self):
 
     def test_echo(self):
         body = 'Hello World!'
-        resp = self.app.get('/echo', data=body)
+        resp = self.app.get('/v1/echo', data=body)
```

Comment:
I like that we now specify API version in tests - makes it more transparent which version of the API we are testing, and makes it possible to test multiple versions (hopefully sometime far in the future)